build/ppas.sh

```
@@ -0,0 +1,72 @@
+#!/bin/sh
+DoExitAsm ()
+{ echo "An error occurred while assembling $1"; exit 1; }
+DoExitLink ()
+{ echo "An error occurred while linking $1"; exit 1; }
+echo Assembling goccia.builtins.semver
+/Library/Developer/CommandLineTools/usr/bin/clang -x assembler -c -target arm64-apple-macosx11.0.0 -o build/Goccia.Builtins.Semver.o  -x assembler build/Goccia.Builtins.Semver.s
+if [ $? != 0 ]; then DoExitAsm goccia.builtins.semver; fi
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)